### PR TITLE
feat(relay-monitor): Validator Registrations

### PR DESCRIPTION
**Pull request by samcm**

### CODE REVIEW

File: pkg/relaymonitor/ethereum/ethereum.go [hunk 1]
```diff
@@ -2,75 +2,91 @@ package ethereum
...
 }
 
-func (b *BeaconNetwork) Start(ctx context.Context) error {
-	// Fetch the network config from the URL
-	networkConfig, err := b.fetchNetworkConfig(ctx)
-	if err != nil {
-		return errors.Wrap(err, "failed to fetch network config")
-	}
+func (b *BeaconNode) Start(ctx context.Context) error {
+	// Start a 5min deadling timer to wait for the beacon node to be ready
```
Comment (mattevans) [inline pkg/relaymonitor/ethereum/ethereum.go:58]: Niceeeee 👌

File: pkg/relaymonitor/registrations/config.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package registrations
+
+import (
+	"github.com/ethpandaops/beacon/pkg/human"
+	"github.com/pkg/errors"
+)
+
+type Config struct {
+	// Enabled is the flag to enable the validator registration monitor
+	Enabled *bool `yaml:"enabled" default:"false"`
```
Comment (mattevans) [inline pkg/relaymonitor/registrations/config.go:10]: Does we need it to a pointer if we default to false? Would avoid dereferencing when using?

Comment (samcm) [reply pkg/relaymonitor/registrations/config.go:10]: I can't remember the exact reason off the top of my head, but it's something to do with the yaml defaults library we're using. I think if you have `default: true` it becomes impossible to set it to `false` in yaml without using a pointer. We're defaulting to `false` here, but we've generally just kept them as pointers to save potential headache if we flip it in the future. Good catch tho

Comment (mattevans) [reply pkg/relaymonitor/registrations/config.go:10]: Ahhhhh yep makes sense. Tyty

**Pull request merged**